docs/contributing/contributing-rules.md

```diff
@@ -0,0 +1,136 @@
+---
+slug: contributing-to-semgrep-rules-repository
+description: "This article outlines how to contribute to Semgrep rules repository.."
+---
+
+# Contributing to Semgrep rules
+
+## Introduction
+
+There are two ways in which you can contribute to the Semgrep rules repository:
+
+<dl>
+    <dt>For users of Semgrep App</dt>
+    <dd>Contribute to the Semgrep rules repository through Semgrep App. This workflow is recommended. See <a href="#contributing-through-semgrep-app-recommended"> Contributing through Semgrep App (recommended)</a>. This workflow creates the necessary pull request for you and streamlines the whole process.
+</dd>
+    <dt>For contributors to the repository through GitHub</dt>
+    <dd>Contribute to the Semgrep rules repository through a pull request. See the <a href="#contributing-through-github"> Contributing through GitHub</a> section for detailed information.</dd>
+</dl>
+
+### Contributing through Semgrep App (recommended)
+
+To contribute to the Semgrep rules repository through Semgrep App, follow these steps:
+1. Go to [Semgrep App Editor](https://semgrep.dev/orgs/-/editor).
+2. Click **Create New Rule**.
+3. Make one of the following steps:
+    - Create a new rule and test code, and then click **Save**. Note: The test file must contain at least one true positive and one true negative test case in order to get approved
```

Comment:
nit: add a link to the section on adding test cases

Reply:
Done!